### PR TITLE
test: add tests

### PR DESCRIPTION
**Pull request by Lehren**
add warning message tests for wave-packerSpec to
increase line coverage to the minimum 5%

### CODE REVIEW

File: test/wave-packerSpec.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+/* eslint-disable
```
Comment (remcohaszing) [inline test/wave-packerSpec.js]: This is a totally new file. Don't add eslint-disable in new files.

File: test/wave-packerSpec.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ camelcase,
+ new-cap
+ */
+
+/* global
```
Comment (remcohaszing) [inline test/wave-packerSpec.js]: Instead of globals, you can just use `env`.

In `.eslintrc` add `jasmine: true`.

File: test/wave-packerSpec.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+const WavePacker = require('../wave-packer');
+
+describe('Warning tests', function() {
+  it('Should warn when the sample rates are invalid', function() {
+    spyOn(console, 'warn');
+    var warning = '48000 or 44100 are the only supported recordingSampleRates';
+    var wavePacker = new WavePacker();
+    wavePacker.init(0, 0, 0);
+    expect(console.warn).toHaveBeenCalledWith(warning);
+    expect(wavePacker.recordingSampleRate).toBeDefined();
```
Comment (remcohaszing) [inline test/wave-packerSpec.js]: Don't just expect it to be defined. Expect it to be the actual value that's used.

Comment (remcohaszing) [inline test/wave-packerSpec.js]: That goes for all occurrences.

File: test/wave-packerSpec.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+ expect,
+ it,
+ spyOn,
+ window,
+ FormData
+ */
+
+const WavePacker = require('../wave-packer');
+
+describe('Warning tests', function() {
```
Comment (remcohaszing) [inline test/wave-packerSpec.js]: Let's name it after what's being tested. In this case `'WavePacker'`.

**Comment by Lehren** [comment]
Integrated changes in latest commit

**Pull request merged**